envoy/upstream/load_balancer.h

```diff
@@ -273,11 +273,6 @@ class LoadBalancerConfig {
 };
 using LoadBalancerConfigPtr = std::unique_ptr<LoadBalancerConfig>;
 
-/**
```

Comment:
FWIW I think it's fine to keep this around to help folks pulling from main, but also not a horribly bad search and replace.